### PR TITLE
Document using cicd built Docker Images

### PR DESCRIPTION
**Pull request by feikesteenbergen**

### CODE REVIEW

**Review by sgichohi — APPROVED** [review]
Nice!

File: README.md [hunk 1]
```diff
@@ -40,6 +40,51 @@ POSTGIS_VERSIONS="" make build
...
+Amazon Elastic Container Registry (ECR).
+
+## Find the image tag for your commit
+
+> Replace *** with the AWS Account ID.
+
+Once your commit is pushed, a Docker Image will be built, and if successful, will be pushed.
+The tag of this Docker Image will be `cicd-<first 8 chars of commit sha>`, for example, for commit `baddcafe...`, the tag will look like:
+
+    ***.dkr.ecr.us-east-1.amazonaws.com/timescaledb-ha:cicd-deadbeef
```
Comment (feikesteenbergen) [inline README.md:55]:
```suggestion
    ***.dkr.ecr.us-east-1.amazonaws.com/timescaledb-ha:cicd-baddcafe
```

Comment (alexeyklyukin) [reply README.md:55]: What are those wildcards in `***.dkr.ecr.us-east-1.amazonaws.com`, is this a template to fill in with the actual hostname?

**Review by alexeyklyukin — APPROVED** [review]
Very nice, thank you!

File: README.md (hunk 1 above)
Comment (feikesteenbergen) [reply README.md:55]: It's our private AWS account id. Although not secret, it is advised to keep it private, background:

https://security.stackexchange.com/questions/192539/keeping-aws-account-id-secret

Comment (feikesteenbergen) [reply README.md:55]: Perhaps being a bit too cautious though, however, once a value is leaked, it cannot be unleaked ...

Comment (alexeyklyukin) [reply README.md:55]: I agree, thank you for making it clear.

**Pull request merged**